example/src/main/java/com/github/appintro/example/util/AppIntroFragmentFactory.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.appintro.example.util
+
+import android.graphics.Color
+import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+import androidx.annotation.FontRes
+import com.github.paolorotolo.appintro.AppIntroFragment
+import com.github.paolorotolo.appintro.model.SliderPage
+
+object AppIntroFragmentFactory {
+    fun makeFragment(title: String, desc: String,
+                     @DrawableRes drawable: Int,
+                     @ColorInt background: Int = Color.TRANSPARENT,
+                     @FontRes titleFont: Int? = null,
+                     @FontRes descFont: Int? = null,
+                     @DrawableRes backgroundDrawable: Int? = null,
+                     titleTypeface: String? = null,
+                     descTypeface: String? = null): AppIntroFragment =
+            AppIntroFragment.newInstance(SliderPage().apply {
+                this.title = title
+                this.description = desc
+                this.imageDrawable = drawable
+                this.bgColor = background
+
+                titleTypeface?.let { this.titleTypeface = it}
+                descTypeface?.let { this.descTypeface = it}
+
+                titleFont?.let { titleTypefaceFontRes = it }
+                descFont?.let { descTypefaceFontRes = it }
+
+                backgroundDrawable?.let { bgDrawable = it }
+            })
+}
```

Comment:
Please remove this class. Ideally you should be able to use one of the overload of `AppIntroFragment.newInstance`.

I would love to avoid the scenario where library developers copy-n-paste code from the various example and their code doesn't compile as `AppIntroFragmentFactory` is not found (given that we have it in the sample app).